packages/react/src/useActionForm.ts

```diff
@@ -336,14 +575,23 @@ export const useActionForm = <
 
       await handleSubmit(
         async (data) => {
+          if (isModelAction) {
+            try {
+              if (!action.hasAmbiguousIdentifier && findResult.data) {
+                data = disambiguateDefaultValues(data, findResult.data, action);
+              }
+
+              data = await reshapeDataForGraphqlApi(api, defaultValues, data);
+            } catch (e: any) {
+              handleSubmissionError(e);
```

Comment:
Somewhat related to the code around here, I've caught my self a few times missing important errors that are produced by useActionForm since I didn't hook into the `onError` function.

If there's ever issues with the `reshapeDataForGraphqlApi` it'll propagate the errors to the `onError` (I kind of assumed it's reserved for errors related to the API) hook. Maybe this isn't the best exeperience for dev's and instead we should just `console.error`?

Reply:
why catch at all? this feels like something that should throw